### DISCUSSION TITLE
Tem algum código no repositório para extrair listas de qualquer site?

### DISCUSSION THREAD

**Discussion by filipeas** [joaomota59/diversos#11] [General] [answered]
Havia um pequeno projeto em particular para fazer extração das notícias dos sites da UFPI e da UESPI, e guardá-las em um banco de dados. Tem algum código nesse repositório que faça isso?

**Comment by joaomota59** [comment, marked as answer]
No momento tem-se o [BS4 IN JS PAGE.py](https://github.com/joaomota59/diversos/blob/main/BS4%20IN%20JS%20PAGE.py) , toda via, já está defasado.. 
Atualmente, é interessante utilizar BeautifulSoup(Caso não tenha muito JS na página) ou Selenium caso haja muita renderização com JS. 
Nisso, é importante estudar o comportamento de cada página e ver qual o melhor para se usar.
